src/modules/checkIsValidUserInput.js

```diff
@@ -8,8 +8,48 @@
  * @param {string} userInput - The user input
  * @return {boolean} - True if the user input is valid, false otherwise
  */
+const checkIsUniqueNumbers = (value) => {
+  const arrayValue = `${value}`.split('');
+
+  if (arrayValue.length !== 4 && typeof value !== 'number') {
```

Comment:
The condition here should use '||' instead of '&&'. It should check if the length is not 4 or if the type is not a number.